src/test/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/volume/v4_9/CloudStackVolumePluginTest.java

```diff
@@ -62,29 +66,32 @@
     private static final String JSON_FORMAT = "json";
     private static final String RESPONSE_KEY = "response";
     private static final String COMMAND_KEY = "command";
-    private static final String DISK_OFFERING_ID_KEY = CreateVolumeRequest.DISK_OFFERING_ID;
-    private static final String NAME_KEY = CreateVolumeRequest.VOLUME_NAME;
-    private static final String SIZE_KEY = CreateVolumeRequest.VOLUME_SIZE;
-    private static final String ZONE_ID_KEY = CreateVolumeRequest.ZONE_ID;
+    private static final String DISK_OFFERING_ID_KEY = DISK_OFFERING_ID_KEY_JSON;
+    private static final String NAME_KEY = NAME_KEY_JSON;
+    private static final String SIZE_KEY = SIZE_KEY_JSON;
+    private static final String ZONE_ID_KEY = ZONE_ID_KEY_JSON;
```

Comment:
Wouldn't it be more effective to remove these private constants from this class using only the external constants?

Reply:
This is legacy code. It was affected due to the other class modification. 
It does not exist anymore. You can see it in the fourth PR.